javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/ApplicationLoggerBridge.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.bootstrap;
+
+import java.util.concurrent.atomic.AtomicReference;
+
+public abstract class ApplicationLoggerBridge {
+
+  private static final AtomicReference<ApplicationLoggerBridge> applicationLoggerBridge =
+      new AtomicReference<>();
+
+  public static void set(ApplicationLoggerBridge bridge) {
+    ApplicationLoggerBridge.applicationLoggerBridge.compareAndSet(null, bridge);
```

Comment:
This doesn't need to be qualified, right?
```suggestion
   applicationLoggerBridge.compareAndSet(null, bridge);
```